config/checkstyle-input-suppressions.xml

```diff
@@ -7841,16 +7841,6 @@
             files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]packageinfo[\\/]test1[\\/]package-info.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]packageinfo[\\/]test2[\\/]package-info.java"/>
-  <suppress id="UnnecessaryOkComment"
```

Comment:
Please show `grep` or some other command line search utility for `genericwhitespace` in this file.

Reply:
done.

Reply:
Use https://github.com/checkstyle/checkstyle/pull/13609#issue-1866293244 as an example, always show "before" and "after", I think you will find a surprise :)

Reply:
done